### ISSUE TITLE
Build view petition page

### ISSUE THREAD

**Issue by christopherlim98** [opengovsg/petitionsgovsg#21]
View for 

- A single petition

Sample mock petition data:

- title: Petition for a 3-day work week
- Which ministry is most relevant to your petition: Ministry of Health
- full name: Alwyn Khaleedah
- additional_info: changemakers at OGP
- what would you like to see changed: 
```
The coronavirus epidemic has shown that we do not need to work 5 days all week.

Data from [test_link](https://www.open.gov.sg/) shows that **we do not need** to _do this_
because of 
1. reason one
2. and two
3. and three

And this will have the following benefits:
- this
- that
- those

```
- why do you want this? 
```
because of 
1. reason one
2. and two
3. and three
```
- user_id: 1
- references: ['wiki.com/worklife', 'wiki.com/balance']
- petition_status: "active"
- petition_created: <date>
- signature_count: 1435

**Comment by aidenywl** [comment]
edited issue to restructure data fields, add rich text data, `additional_info`, `petition_created`, `petition_status` and collapsed summary into "what would you like to see changed?

**Comment by christopherlim98** [comment]
Closed by #76
